### ISSUE TITLE
bin/dev commands not available

### ISSUE THREAD

**Issue by pquerner** [JeroenBoersma/docker-compose-development#72]
On my Mac Sierra I cannot get the following commands to work:
build
rebuild

They report this issue:

```
./bin/dev build
./bin/dev: line 45: cd: /Users/pq/development/bin/dev_command/bin/dev_command: No such file or directory
ERROR: .IOError: [Errno 2] No such file or directory: u'/Users/pq/development/bin/dev_command/docker-compose-osx.yml'
```

**Comment by JeroenBoersma** [comment]
From what I can tell, your docker-compose looks outdated, can you check this?

**Comment by pquerner** [comment]
```
docker-compose --version
docker-compose version 1.18.0, build 8dd22a9
```

1.19 is not yet availble. Its RC currently. https://github.com/docker/compose/tags

**Comment by JeroenBoersma** [comment]
strange, thanks for this feedback.

1.18 is new enough. I'm unable to reproduce this...
I know this was a problem in earlier versions.

the `bin/dev_command/bin/dev_command` double `bin/dev_command` is the problem, but I cannot tell exactly why yet.

**Comment by JeroenBoersma** [comment]
ah, I've a clue, `build` and `rebuild` will start some dev commands themselves...

**Comment by JeroenBoersma** [comment]
Okay, should be resolved in latest push 8d265c4e0e2f4445ab6256871995aaa88db0329f

**Comment by pquerner** [comment]
Confirming that its working now. Thanks.
